Building with WITH_PROVIDERS=OFF fails for v2.5.0

### Expected behaviour
Setting -DWITH_PROVIDERS=OFF for cmake should configure the client without support for pre-defined providers.

### Actual behaviour
Build fails with
```
src/gui/wizard/owncloudsetuppage.cpp: In constructor ‘OCC::OwncloudSetupPage::OwncloudSetupPage(QWidget*)’:
src/gui/wizard/owncloudsetuppage.cpp:95:9: error: ‘class Ui_OwncloudSetupPage’ has no member named ‘slideImage’; did you mean ‘slideShow’?
     _ui.slideImage->hide();
         ^~~~~~~~~~
         slideShow
src/gui/wizard/owncloudsetuppage.cpp:96:9: error: ‘class Ui_OwncloudSetupPage’ has no member named ‘slideLabel’; did you mean ‘topLabel’?
     _ui.slideLabel->hide();
         ^~~~~~~~~~
         topLabel
```

### Steps to reproduce
1. Use cmake with -DWITH_PROVIDERS=OFF
2. cmake --build . --target all

### Client configuration
Client version: 2.5.0

Operating system: All

## Note
Please consider extending your CI builds with more configuration options.

--- a/src/gui/wizard/owncloudsetuppage.cpp
+++ b/src/gui/wizard/owncloudsetuppage.cpp
@@ -92,10 +92,9 @@ OwncloudSetupPage::OwncloudSetupPage(QWidget *parent)
     _ui.slideShow->setPalette(pal);
 #else
     _ui.createAccountButton->hide();
-    _ui.slideImage->hide();
-    _ui.slideLabel->hide();
     _ui.loginButton->hide();
     _ui.installLink->hide();
+    _ui.slideShow->hide();
 #endif
     setStyleSheet(QString("background-color:%1; color:%2 QLabel { color:%2; } QSpacerItem { color: red; }").arg(theme->wizardHeaderBackgroundColor().name(), theme->wizardHeaderTitleColor().name()));
 }

